README.md

```diff
@@ -258,6 +273,7 @@ Arguments:
 - name: The name of the publication
 - collection: A function that returns the collection to publish (or just a collection, if it's insecure).
 - selector: (Optional) A function that returns mongo selector that will limit the results published (or just the selector).
+- settings: (Optional) A object with settings on server side's publish function. (Details see below)
```

Comment:
You're right, there is a vulnerability - thanks. There should be a way to limit fields on the server. I'm afraid extending options won't be enough though. There's also fields from the client used to create the filter query, so an attacker could pass ['secret'] as the fields with some value as the filter and find out which documents have that value for it, even if it's not returned.

This is a separate issue from enableRegex though.